problem_set_1.py

```diff
@@ -91,3 +127,20 @@
 
 # Write a function called `gcd` that takes two arguments and returns the greatest
 # common divisor using the instructions above.
+def gcd(a,b):
+    d = 0
+    while (a%2 ==0 and b%2 ==0):
```

Comment:
This is great work. Can you see places where previous functions you wrote could be of use?